css/src/components/table.css

```diff
@@ -0,0 +1,168 @@
+.yoast table {
+	font-size: var(--font-size-default);
+	color: var(--color-default);
+	line-height: 1.2;
+	width: 100%;
+	border-bottom: var(--border-default);
+	border-spacing: 0;
```

Comment:
`border-spacing: 0;` and `width: 100%;` are already set by the WordPress CSS via the `.widefat` class. Any reason why we want to repeat these properties here?